doc/sphinx/userguide/faq.md

```diff
@@ -1,12 +1,227 @@
 # Frequently Asked Questions
 
-```{caution}
-This page is a work in progress. Sorry.
+```{hint}
+If your question isn’t answered here, consider asking us on the
+[Cantera Users' Group](https://groups.google.com/g/cantera-users).
 ```
 
 (sec-faq-installation)=
 ## Installation
 
+:::{dropdown} How do I determine the cause of the error "DLL load failed while importing _cantera: The specified module can not be found."?
+:color: primary
+
+1. Install the [Dependencies](https://github.com/lucasg/Dependencies) tool.
+2. *If you are using Conda*: Run this tool from a terminal where your Cantera Conda
+   environment is active. That is, run the command:
+   ```
+   start "C:\Program Files\Dependencies\DependenciesGUI.exe"
+   ```
+   from that terminal, replacing the path with the path to `DependenciesGUI.exe` on your
+   computer. This step is necessary to have the `PATH` environment variable set
+   correctly.
+3. Use Dependencies to open the Cantera Python extension module. If you installed
+   Mambaforge at `C:\mambaforge` and Cantera in an environment named `ct-env`, then this
+   file would be named something like
+   `C:\mambaforge\envs\ct-env\Lib\site-packages\cantera\_cantera.cp312-win_amd64.pyd`.
+4. Identify any DLLs that the tool is unable to find, or if there are methods that are
+   not being resolved. One thing to keep an eye out for is if there are any libraries
+   that are being loaded from unexpected locations (for example, other than from your
+   conda installation and the Windows system directories).
+:::
+
 ## Input Files
-* Where can I find chemical mechanisms to use with Cantera?
-* How do I fix errors converting Chemkin input files to Cantera's format?
+
+:::{dropdown} Where can I find chemical mechanisms to use with Cantera?
```

Comment:
Is it worth adding a note here that these are not endorsed by the Cantera developers and are used at their own risk?